web/src/styles/scss/tables.scss

```diff
@@ -1,33 +1,70 @@
 @import '@cmsgov/design-system-support/src/settings/index';
 
-.table-cms {
+.budget-table {
   font-size: 12px;
-  &.table-cms__totals {
-    width: 42%; // roughly equal to full tables' cells; temp fix before switching font to courier
+  &.budget-table--totals {
+    width: 50%;
   }
-  &.centered-headers th {
-    text-align: center;
+  + .budget-table {
+    margin-top: $spacer-2;
   }
+
   th {
-    background-color: #DAE9F7;
-    font-weight: normal;
+    font-weight: $font-semibold;
   }
-  td {
-    background-color: #F3F8FC;
-  }
-  th,
-  td {
-    border-left: 4px solid $color-white;
-    border-top: 4px solid $color-white;
-    &:first-child {
-      border-left: none;
-    }
 
+  th, tr {
+    &.budget-table--total,
+    &.budget-table--subtotal,
+    &.budget-table--sum {
+      font-weight: $font-bold;
+      th, td {
+        font-weight: $font-bold;
+      }
+    }
   }
-  + .table-cms:not(.table-frozen-data){
-    margin-top: $spacer-2;
+
+  thead {
+    th {
+      background-color: $color-gray-lighter;
+      white-space: pre-line;
+      /* used for highlights in a column */
+      &.budget-table--col__highlight {
+        background-color: $color-warn-lightest;
+      }
+    }
+    border-top: 1px solid $color-gray-light;
+    border-bottom: 1px solid $color-gray-light;
```

Comment:
Do all these colors come from the design system?  I'm just a little confused because I know ace-css brings in some colors too, but I guess those are classes, not variables.

Reply:
Yesssss, these are all the design system colors! Good gut check!